src/api/app/controllers/webui/package_controller.rb

```diff
@@ -964,7 +964,7 @@ def rpmlint_log
       @log = get_rpmlint_log(params[:project], params[:package], params[:repository], params[:architecture])
       @log.encode!(xml: :text)
       render partial: 'rpmlint_log'
-    rescue ActiveXML::Transport::NotFoundError
+    rescue ActiveXML::Transport::NotFoundError, ActiveXML::Transport::Error
```

Comment:
Hm, I'm not sure if the message "No rpmlint log" is correct in this case?
Do you have an example errbit entry?

Reply:
Found it in the bug...

So the exception is raised because the URL is wrong and not because the rpmlint log does not exist. So we should not catch it in my opinion and should fix the issue instead!